app/src/main/res/layout/commit_diff_line.xml

```diff
@@ -18,7 +18,7 @@
     android:id="@+id/tv_diff"
     style="@style/CodeText"
     android:layout_width="match_parent"
-    android:background="@drawable/list_item_background"
```

Comment:
@hzsweers You've replaced many references to this drawable. Should the drawable itself be removed too, or are there still references to the drawable left?